docs/developer-docs/latest/developer-resources/database-apis-reference/entity-service/crud.md

```diff
@@ -1,10 +1,8 @@
 ---
 title: CRUD operations with Entity Service API - Strapi Developer Docs
```

Comment:
I see that for this one the title end with Strapi Developer Docs. It requires fewer characters (max should be 60). Do you think we can do that for every title?